packages/loader/driver-utils/src/blobToString.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IsoBuffer } from "@fluidframework/common-utils";
+
+/**
+ * Convert binary blob to string format
+ *
+ * @param blob - the binary blob
+ * @returns the blob in string format
+ */
+export function blobToString(blob: ArrayBufferLike): string {
+    const content = IsoBuffer.from(blob, "utf8").toString("utf8");
```

Comment:
you can just return result without temporarily variable, and compiler will force you to replace it to
export const blobToString = (blob: ArrayBufferLike) => IsoBuffer.from(blob, "utf8").toString("utf8");

Reply:
same below